blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/PreaggregateConversions.java

```diff
@@ -129,10 +130,12 @@
         List<PreaggregatedMetric> list = new ArrayList<PreaggregatedMetric>(enums.size());
         for (BluefloodEnum en : enums) {
             Locator locator = Locator.createLocatorFromPathComponents(tenant, en.getName().split(NAME_DELIMITER, -1));
-            BluefloodEnumRollup rollup = new BluefloodEnumRollup();
-            rollup = rollup.withEnumValue(en.getValue(), 1L);
-            PreaggregatedMetric metric = new PreaggregatedMetric(timestamp, locator, DEFAULT_TTL, rollup);
-            list.add(metric);
+            if (!ExcessEnumReader.getInstance().isInExcessEnumMetrics(locator)) {
+                BluefloodEnumRollup rollup = new BluefloodEnumRollup();
+                rollup = rollup.withEnumValue(en.getValue(), 1L);
+                PreaggregatedMetric metric = new PreaggregatedMetric(timestamp, locator, DEFAULT_TTL, rollup);
+                list.add(metric);
+            }
```

Comment:
Where are we actually returning an error to the user and/or writing errors to a log that the metrics ingestions were rejected because it had reached the bad metrics count (determined by the isInExcessEnumMetrics call)?